ts/Dashboards/Components/DataGridComponent/DataGridComponent.ts

```diff
@@ -160,8 +151,11 @@ class DataGridComponent extends Component {
         this.innerResizeTimeouts = [];
 
 
-        this.on('afterSetConnector', (e: any): void => {
-            this.disableEditingModifiedColumns(e.connector);
+        this.on('afterSetConnectors', (e: any): void => {
+            const connector = e.connectorHandlers?.[0]?.connector;
```

Comment:
The `connectorHandler` is always an array (default definition in the `Component.ts`), so what is the use case of missing the array type?